RPG-game-Hung/classes/actions.py

```diff
@@ -0,0 +1,27 @@
+from termcolor import colored
+
+from classes.colors import Color
+
+
+class Action:
```

Comment:
`class` should also include `DocString` to document what this class for, how it works, or if there is something user should pay attention when using it, etc.